weather10kb.js

```diff
@@ -25,18 +25,21 @@ function Weather10kbRequest(request) {
           apiKey: process.env.GOOGLE_API_KEY,
           formatter: null         // 'gpx', 'string', ...
         });
+        var countryCode = 'US';
```

Comment:
A nice enhancement could be to default to the language from the country of origin.

todo/ change html lang="EN" …